recipes/sqlite3/all/CMakeLists.txt

```diff
@@ -21,6 +21,9 @@ option(ENABLE_UNLOCK_NOTIFY "Enable support for the unlock notify API")
 option(ENABLE_DEFAULT_SECURE_DELETE "Turns on secure deletion by default")
 option(USE_ALLOCA "The alloca() memory allocator will be used in a few situations where it is appropriate.")
 option(OMIT_LOAD_EXTENSION "Omits the entire extension loading mechanism from SQLite")
+if(SQLITE3_VERSION VERSION_GREATER_EQUAL "3.35.0")
```

Comment:
VERSION_GREATER_EQUAL  requires CMake 3.7, may you add check?

Reply:
good catch, let's bump min version to 3.7 (4 years old version, sounds reasonable)